recipes/configure-ca-webapp-nginx-proxy.rb

```diff
@@ -0,0 +1,27 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: configure-ca-webapp-nginx-proxy
+
+include_recipe "mh-opsworks-recipes::update-package-repo"
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+ca_webapp_info = node.fetch(:ca_webapp, {})
+app_name = ca_webapp_info.fetch(:ca_webapp_name, "webapp")
+
+install_package("nginx")
+
+install_nginx_logrotate_customizations
+
+ssl_info = node.fetch(:ssl, get_dummy_cert)
+if cert_defined(ssl_info)
+  create_ssl_cert(ssl_info)
+  certificate_exists = true
+end
+
+template %Q|/etc/nginx/sites-enabled/default| do
```

Comment:
You can see this in the `recipes/configure-nginx-proxy.rb` and `recipes/update-nginx-config-for-ganglia.rb` recipes.  It might be nice to leverage this here, then we can have multiple services with frontend / backend proxies living harmoniously on this machine.